src/vivarium_csu_zenon/data/loader.py

```diff
@@ -514,6 +412,93 @@ def load_standard_excess_mortality_rate(key: str, location: str) -> pd.DataFrame
     return _load_em_from_meid(meids[key], location)
 
 
+def load_ikf_exposure(key: str, location: str) -> pd.DataFrame:
+    key = EntityKey(key)
+    entity = get_entity(key)
+
+    location_id = utility_data.get_location_id(location) if isinstance(location, str) else location
+    measure = 'exposure'
+    raw_validation.check_metadata(entity, measure)
+
+    data = gbd.get_exposure(entity.gbd_id, location_id)
+    data = normalize_ikf_exposure_distribution(data)
+    raw_validation.validate_raw_data(data, entity, measure, location_id)
+
+    data = data.drop('modelable_entity_id', 'columns')
+
+    data = utilities.filter_data_by_restrictions(data, entity, 'outer', utility_data.get_age_group_ids())
+
+    tmrel_cat = utility_data.get_tmrel_category(entity)
+    exposed = data[data.parameter != tmrel_cat]
+    unexposed = data[data.parameter == tmrel_cat]
+
+    #  FIXME: We fill 1 as exposure of tmrel category, which is not correct.
+    data = pd.concat([utilities.normalize(exposed, fill_value=0), utilities.normalize(unexposed, fill_value=1)],
+                     ignore_index=True)
+
+    # normalize so all categories sum to 1
+    cols = list(set(data.columns).difference(vi_globals.DRAW_COLUMNS + ['parameter']))
+    sums = data.groupby(cols)[vi_globals.DRAW_COLUMNS].sum()
+    data = (data
+            .groupby('parameter')
```

Comment:
@rmudambi my change here was a formatting one.  This all should be a no-op, so you can definitely drop if you want.